src/django_mysql/models/fields/datetime.py

```diff
@@ -0,0 +1,78 @@
+from django.core import checks
+from django.db.models import DateTimeField as DjangoDateTimeField
+from django.db.models.fields import DateTimeCheckMixin as DjangoDateTimeCheckMixin
+from django.utils import timezone
+
+
+class DateTimeCheckMixin(DjangoDateTimeCheckMixin):
```

Comment:
There's no need to subclass the mixin to replace this method in the new `DateTimeField` class.